doc/develop/test/ztest.rst

```diff
@@ -351,6 +351,21 @@ efforts into the specific module in question. This will speed up testing since
 only the module will have to be compiled in, and the tested functions will be
 called directly.
 
+Examples of unit tests can be found in ``tests/unit/``. In the following
```

Comment:
looks like this is missing some kind of transition between the two sentences and an actual explanation of what needs to be done rather than "here's an example". Something like the below?

> In order to declare the unit tests present in a source folder, you need to add the relevant source files to the ``testbinary`` target from the :zephyr_file:`cmake/modules/unittest.cmake <unittest>` component. See a minimal example below: 
> ...
```